x/staking/keeper/sdk_types.go

```diff
@@ -142,3 +142,17 @@ func (k Keeper) IterateDelegations(ctx sdk.Context, delAddr sdk.AccAddress,
 		i++
 	}
 }
+
+// return all delegations used during genesis dump
```

Comment:
ditto strange name for this file as well -- based on the contents.

Reply:
similarly to my previous comment - this filename as well as the previous one (`for_expected_keepers.go`) should be renamed something consistent... don't know what though

Reply:
I've updated to the name `alias_functions.go` best name I can think of at the moment